salt/metalk8s/kubeadm/init/certs/front-proxy-ca.sls

```diff
@@ -0,0 +1,77 @@
+{%- from "metalk8s/map.jinja" import ca with context %}
```

Comment:
I would prefer to have those information in `front_proxy.ca` even if it's duplicated it's 2 different CA IMO

Reply:
I will create an entry for front_proxy_ca in the default.yaml then